faqs/common-errors-faq.md

```diff
@@ -77,7 +76,40 @@ An example: For `POST:/accesstoken/get` the error may be like this if you send a
 
 ## Why do I get `HTTP 401 Unauthorized`?
 
-This means you are using the wrong API keys or Authorization header.
+If you get errors like below from Microsoft Azure, this means you are using the wrong API keys or Authorization header:
+
+```json
+{
+  "error": "unauthorized_client",
+  "error_description":
+    "AADSTS70001: Application with identifier 'e9b6c99d-2442-4a5d-84a2-\
+     c53a807fe0c4' was not found in the directory testapivipps.no\
+     Trace ID: 3bc2b2a0-d9bb-4c2e-8367- 5633866f1300\r\nCorrelation ID:\
+     bb2f4093-70af-446a-a26d-ed8becca1a1a\r\nTimestamp: 2017-05-19 09:21:28Z",
+  "error_codes": [ 70001 ],
+  "timestamp": "2017-05-19 09:21:28Z",
+  "trace_id": "3bc2b2a0-d9bb-4c2e-8367-5633866f1300",
+  "correlation_id": "bb2f4093-70af-446a-a26d-ed8becca1a1a"
+}
+```
+
+```json
+{
+  "error":"unauthorized_client",
+  "error_description":
+    "AADSTS700016: Application with identifier \'my_client_id\'
+     was not found in the directory \'tenant_directory\'.
+     This can happen if the application has not been installed
+     by the administrator of the tenant or consented to by any
+     user in the tenant. You may have sent your authentication
+     request to the wrong tenant.",
+  "error_codes":[700016],
+  "timestamp":"2021-03-23 06:46:31Z",
+  "trace_id":"<snip>",
+  "correlation_id":"<snip>",
+  "error_uri":"https://login.windows.net/error?code=700016"
+}
```

Comment:
I think merging these is wrong, the `unauthorized_client` case is for a very specific scenario (using invalid client_id for the Get Accesstoken request). So it doesn't really work as an general example json, so I would say it's better to remove any information about `unauthorized_client` or keep it as an specific FAQ answer.

Reply:
If you want to add json example response here, you should probably add our RFC error response, since that's the most generic one we have (even though not many services have actually implemented it)